pom.xml

```diff
@@ -12,8 +12,8 @@
     <description>Offers matrix-based security authorization strategies (global and per-project).</description>
     <url>http://wiki.jenkins-ci.org/display/JENKINS/Matrix+Authorization+Strategy+Plugin</url>
     <properties>
-        <jenkins.version>1.609.1</jenkins.version>
-        <java.level>6</java.level>
+        <jenkins.version>2.46.1</jenkins.version>
```

Comment:
Java 8 is required only starting from 2.60.1. I would advice against doing such bump 🐛 
Java 7 for this baseline would be fine

Reply:
Just use 2.60.3 and be done with it. That is what [most users are on anyway](http://stats.jenkins.io/pluginversions/matrix-auth.html).